src/Dapr.Actors/Client/ActorProxyOptions.cs

```diff
@@ -26,5 +30,24 @@ public JsonSerializerOptions JsonSerializerOptions
             set => this.jsonSerializerOptions = value ??
                     throw new ArgumentNullException(nameof(JsonSerializerOptions), $"{nameof(ActorProxyOptions)}.{nameof(JsonSerializerOptions)} cannot be null");
         }
+
+        /// <summary>
+        /// The Dapr Api Token that is added to the header for all requests.
+        /// </summary>>
+        public string DaprApiToken
+        {
+            get => this.daprApiToken;
+            set => this.daprApiToken = value;
+        }
+
+        /// <summary>
+        /// The HTTP message handler. Used for testing purpose
+        /// </summary>>
+        public HttpClientHandler HttpClientMessageHandler
+        {
```

Comment:
What's the purpose of adding this? This breaks a lot of the design work we did earlier.